tafrigh/writer.py

```diff
@@ -70,12 +75,25 @@ def write_vtt(
     ) -> None:
         self._write_to_file(file_path, self.generate_vtt(segments))
 
+    def write_csv(
+        self,
+        file_path: str,
+        segments: list[dict[str, Union[str, float]]],
+        delimiter=',',
+    ) -> None:
+        with open(file_path, 'w', encoding='utf-8') as fp:
+            writer = csv.writer(fp, delimiter=delimiter)
+            writer.writerow(['text', 'start', 'end'])
+
+            for segment in segments:
+                writer.writerow([segment['text'], segment['start'], segment['end']])
```

Comment:
```suggestion
        with open(file_path, 'w', encoding='utf-8') as fp:
            header = ['text', 'start', 'end']
            writer = csv.DictWriter(fp, fieldnames=header, delimiter=delimiter)
            writer.writeheader()
            writer.writerows(segments)
```
Try this.. much cleaner :)

Reply:
المشكلة أن ترتيب الخصائص داخل `segments` يختلف، لا يبدأ بـ `'text'`. هل لديك فكرة؟

Reply:
المفروض الترتيب ياخذه من fieldnames.

Reply:
تم التعديل، جزاك الله خيرا.